tests/suites/main_test.function

```diff
@@ -77,6 +79,12 @@ SUITE_POST_DEP
 /*----------------------------------------------------------------------------*/
 /* Test dispatch code */
 
+#if defined(__unix__) || (defined(__APPLE__) && defined(__MACH__))
+#include <unistd.h>
```

Comment:
I think an additional `includes.function` is unnecessary. All the includes we meed in the main body of `main_test.function` should be included there.